app/models/vm_or_template.rb

```diff
@@ -1923,6 +1923,10 @@ def validate_supported
     {:available => true,   :message => nil}
   end
 
+  def validate_unsupported(message_prefix)
```

Comment:
@roliveri yes, i guess deleting that, since all other methods are here

operations are not used for Template class, so SSA for all templates now throws 500